### PR TITLE
Added conversion functions

### PR DESCRIPTION
**Pull request by ghost**
## Changes introduced with this PR

Implementing the conversion functions.

---
By contributing to this repository, I agree to the [contribution guidelines](https://github.com/arcalot/.github/blob/main/CONTRIBUTING.md).

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	$(MAKE) -C $(LATEX-BW)/ pdf
+
+sample:
+	$(SPHINXBUILD) -b latex -t sample $(SCREENOPTS) $(ALLSPHINXOPTS) $(BUILDDIR)/sample
+	sed -i .bak -f process-latex $(BUILDDIR)/sample/$(LATEX-NAME).tex
+	rsync -a latex/ $(BUILDDIR)/sample/
+	cp figs-pdf/stamp*.pdf $(BUILDDIR)/sample/
+	$(MAKE) -C $(BUILDDIR)/sample pdf
+	# chapter 1, 3, 5
+	pdftk A=$(BUILDDIR)/sample/$(LATEX-NAME).pdf cat A1-6 A21-25 A45-48 A84 output $(BUILDDIR)/sample/$(LATEX-NAME)-sample.pdf
```
Comment (lmilbaum) [inline docs/Makefile:78]: end of line

File: pyproject.toml [hunk 2]
```diff
@@ -1,18 +1,21 @@
...
 kubernetes = ">=25.3.0"
 PyYAML = ">=5.4"
+pylint = "^2.16.2"
+coverage = "^7.2.1"
+html2text = "^2020.1.16"
+sphinxcontrib-apidoc= "^0.3.0"
 
 [build-system]
-requires = ["poetry-core"]
+requires = ["poetry-core>=1.0.0"]
```
Comment (lmilbaum) [inline pyproject.toml:20]: Would suggest pinning to a specific version for reproducible build

**Review by lmilbaum — CHANGES_REQUESTED** [review]
I was not able to review all the PR. Too large for my ADHD.

File: pyproject.toml [hunk 3]
```diff
@@ -1,18 +1,21 @@
 [tool.poetry]
 name = "arcaflow-lib-kubernetes"
-version = "0.1.0"
```
Comment (HubertStefanski) [inline pyproject.toml:3]: Is there a reason to roll the version back to 0.0.0?

**Review by Harshith-umesh — APPROVED** [review]
LGTM!

**Pull request merged**